src/Microsoft.Framework.ConfigurationModel/Configuration.cs

```diff
@@ -8,12 +9,19 @@
 
 namespace Microsoft.Framework.ConfigurationModel
 {
-    public class Configuration : IConfiguration, IConfigurationSourceContainer
+    public class Configuration : IConfiguration, IConfigurationSourceRoot
     {
         private readonly IList<IConfigurationSource> _sources = new List<IConfigurationSource>();
-      
-        public Configuration()
+
+        public Configuration(params IConfigurationSource[] sources)
```

Comment:
From other commit:

I think in C# 6 you can do `IEnumerable<T>` params, right?